FWCore/Utilities/src/FriendlyName.cc

```diff
@@ -16,6 +16,11 @@
 // templated class.  Up front we'd register a class name to a transformation function (which would probably take a std::vector<std::string> which holds
 // the results of the node transformations)
 
+namespace {
+  constexpr bool debug = false;
+  std::string prefix; // used only if debug == true
```

Comment:
This is likely to be flagged by the static analyzer as not being thread safe

Reply:
Good point. Should I then change the `debug` above to a macro and hide the global `prefix` by default? Or do you have better ideas?